token.go

```diff
@@ -34,6 +34,8 @@ type TokenCreateRequest struct {
 type Token struct {
 	KeystoneToken string `json:"token"`
 	ExpiresAt     string `json:"expires_at"`
+	TenantName    string `json:"tenant_name"`
```

Comment:
use ProjectName instead of TenantName, Tenant is deprecated when we support IAM service